### DISCUSSION TITLE
2023/03/20/14.openwrt/

### DISCUSSION THREAD

**Discussion by giscus** [qllokirin/hexo_blog_comments#2] [Announcements]
# 2023/03/20/14.openwrt/

https://qllokirin.github.io/2023/03/20/14.openwrt/

**Comment by zzsqwq** [comment]
之前试过网易云解锁的插件，不过是下的 OpenWrt 固件中集成的，总是和 OpenClash 有些莫名其妙的冲突，而且只有 Windows 可以无感使用，Linux 和 Mac 都要设置下证书才可以，于是为了大多数人的无感体验，还是作罢了。

**Reply by qllokirin** [reply]
> 之前试过网易云解锁的插件，不过是下的 OpenWrt 固件中集成的，总是和 OpenClash 有些莫名其妙的冲突，而且只有 Windows 可以无感使用，Linux 和 Mac 都要设置下证书才可以，于是为了大多数人的无感体验，还是作罢了。

原来如此，总感觉OpenWrt最有用的插件还是openclash，其他的插件感觉都太高级看不懂要么就是很鸡肋，战sir有什么推荐的插件吗

**Reply by zzsqwq** [reply]
我也感觉最有用的就是 OpenClash 了，还有几个感觉蛮有用的几个：

- 动态DNS（DDNS）：对于校园网这种 IP 可能会变化的场景还挺好用的，可以直接域名访问管理后台
- 需要内网穿透的话还有 Zerotier 和 Frp 啥的不错
- 用于端口转发网络管理的 Socat

其他的我感觉用处不大，还有一些固件的黑魔法，多拨啥的，可能也有用，不过我也没试过，听说可以网速跑的很快，两倍甚至三倍

**Reply by qllokirin** [reply]
哈哈 不是听说你和bw的内网穿透然后云服务器的ip被学校ban了，其他几个我梭哈梭哈

**Reply by zzsqwq** [reply]
哈哈哈确实，内网穿透还是别搞了，危
